src/orqviz/scans/scans_1D.py

```diff
@@ -24,7 +29,10 @@ def perform_1D_scan(
 
     Args:
         origin: Parameter vector that is the origin on the 1D scan.
-        loss_function: Loss function to perform the scan on.
+        loss_function: Function to perform the scan on. It must receive only a
+            numpy.ndarray of parameters, and return a real number.
+            If your function requires more arguments, consider using the 'partial'
```

Comment:
[suggestion]: We could use `LossFunctionWrapper`, now that we have it.

Reply:
9b76b54718981db8258fcbb88c83ec55ee4200a9